mon-pix/app/templates/fill-in-campaign-code.hbs

```diff
@@ -46,7 +46,10 @@
 
         {{#if this.isUserAuthenticated}}
           <div class="fill-in-campaign-code__warning">
-            <span>{{this.warningMessage}}</span>&nbsp;<span class="link" {{action this.disconnect}}>{{t 'pages.fill-in-campaign-code.warning-message-logout'}}</span>
+            <span>{{this.warningMessage}}</span>&nbsp;
+            <LinkTo @route="logout" class="link">
```

Comment:
On pase de l'action `this.disconnect` à un lien sur `logout`.
Est-ce que le fonctionnement est bien le même ? Est-ce qu'on supprime la méthode correspondante ?

Reply:
Merci pour la correction,d 'ailleurs il y a une différence fonctionnelle : 
- La redirection sur logout renvoyait sur l'accueil
- L'appel à la méthode te fait rester sur la page campagne

Reply:
Oui je me suis rendu compte de ça après coup.
Il me semblait logique de retourner sur l'écran d'accueil après s'être déconnecté mais apparemment non. :/